svc/pkg/ds-log/ops/read/tests/integration.rs

```diff
@@ -0,0 +1,89 @@
+// use chirp_worker::prelude::*;
```

Comment:
Should these tests be reimplemented?

Reply:
i'm going to get as much as i can up to speed soon